modules/reindex/build.gradle

```diff
@@ -23,6 +25,19 @@ esplugin {
   hasClientJar = true
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
I think we should not do this on modules,  lets just run packaged rest tests on the netty4 module